### PR TITLE
Tables for alg params - QGIS vector create

### PR DESCRIPTION
**Pull request by havatv**
Goal:

Ticket(s): #4429

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/processing_algs/qgis/vectorcreation.rst [hunk 1]
```diff
@@ -294,17 +500,53 @@ The points correspond to the pixel centroids that intersect the polygon layer.
...
+     - Description
+   * - **Raster layer**
+     - ``INPUT_RASTER``
+     - [raster]
+     - Input raster layer
+   * - **Vector layer**
+     - ``INPUT_VECTOR``
+     - [vector: polygon]
+     - Input polygon vector layer
+   * - **Points from polygons**
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorcreation.rst]:
```suggestion
   * - **Points inside polygons**
```

File: source/docs/user_manual/processing_algs/qgis/vectorcreation.rst [hunk 2]
```diff
@@ -294,17 +500,53 @@ The points correspond to the pixel centroids that intersect the polygon layer.
...
+.. list-table::
+   :header-rows: 1
+   :widths: 20 20 20 40
+   :stub-columns: 0
+
+   * - Label
+     - Name
+     - Type
+     - Description
+   * - **Points from polygons**
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorcreation.rst]:
```suggestion
   * - **Points inside polygons**
```

File: source/docs/user_manual/processing_algs/qgis/vectorcreation.rst [hunk 3]
```diff
@@ -324,64 +566,175 @@ if present in the photo, will be added to the point as attributes.
...
+     - Name
+     - Type
+     - Description
+   * - **Input folder**
+     - ``FOLDER``
+     - [folder]
+     - Path to the source folder containing the geotagged photos
+   * - **Scan recursively**
+     - ``RECURSIVE``
+     - [boolean]
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorcreation.rst]: Default?
```suggestion
     - [boolean]

       Default: False
```

File: source/docs/user_manual/processing_algs/qgis/vectorcreation.rst [hunk 4]
```diff
@@ -506,33 +976,72 @@ A minimum distance can be specified, to avoid points being too close to each oth
...
+
+       Default: 0
+     - Sampling strategy to use. One of:
+       
+       * 0 --- Points count: number of points for each feature
+       * 1 --- points density: density of points for each feature
+
+   * - **Number or density of points**
+     - ``VALUE``
+     - [expression]
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorcreation.rst]:
```suggestion
     - [number |dataDefined|]
```
sorry, may not be the right spelling but that's the idea of indicating numeric data-definable parameters

File: source/docs/user_manual/processing_algs/qgis/vectorcreation.rst [hunk 5]
```diff
@@ -506,33 +976,72 @@ A minimum distance can be specified, to avoid points being too close to each oth
...
+     - ``STRATEGY``
+     - [enumeration]
+
+       Default: 0
+     - Sampling strategy to use. One of:
+       
+       * 0 --- Points count: number of points for each feature
+       * 1 --- points density: density of points for each feature
+
+   * - **Number or density of points**
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorcreation.rst]:
```suggestion
   * - **Point count or density**
```

File: source/docs/user_manual/processing_algs/qgis/vectorcreation.rst [hunk 6]
```diff
@@ -506,33 +976,72 @@ A minimum distance can be specified, to avoid points being too close to each oth
...
+     - [expression]
+
+       Default: 1.0
+     - The number or density of points, depending on the chosen
+       ``Sampling strategy``.
+   * - **Minimum distance between points**
+     - ``MIN_DISTANCE``
+     - [number]
+
+       Default: 0.0
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/vectorcreation.rst]: 0 or *not set*?

File: source/docs/user_manual/processing_algs/qgis/vectorcreation.rst [hunk 7]
```diff
@@ -157,50 +244,92 @@ units of this CRS.
...
+   :stub-columns: 0
+
+   * - Label
+     - Name
+     - Type
+     - Description
+   * - **Grid**
+     - ``OUTPUT``
+     - [vector: any]
+     - Resulting vector grid layer
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectorcreation.rst]:
```suggestion
     - Resulting vector grid layer. The output geometry type (point, line or polygon) depends
       on the :guilabel:`Grid type`.
```

File: source/docs/user_manual/processing_algs/qgis/vectorcreation.rst [hunk 8]
```diff
@@ -579,68 +1128,158 @@ Any nodata pixels are skipped in the output.
...
-Creates a new point layer with a given number of regular points, all of them within
-a given extent.
+Creates a new point layer with its points placed in a regular grid
+within a given extent.
 
-Together with the point number. two different sampling strategies can be chosen.
+The grid is specified either by the spacing between the points (same
+spacing for all dimensions) or by the number of points to generate.
+In the latter case, the spacing will be determined from the extent.
+In order to generate a full square grid, at least the number of
```
Comment (havatv) [inline source/docs/user_manual/processing_algs/qgis/vectorcreation.rst]:
```suggestion
In order to generate a full rectangular grid, at least the number of
```

File: source/docs/user_manual/processing_algs/qgis/vectorcreation.rst [hunk 9]
```diff
@@ -579,68 +1126,158 @@ Any nodata pixels are skipped in the output.
...
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/vectorcreation.rst]: Just to make the user aware of the fact that the number of generated points can be larger than the number of points specified (to generate a rectangular grid).

File: source/docs/user_manual/processing_algs/qgis/vectorcreation.rst [hunk 10]
```diff
@@ -579,68 +1126,158 @@ Any nodata pixels are skipped in the output.
...
+
+       Default: 100
+     - Spacing between the points, or the number of points, depending
+       on whether ``Use point spacing`` is checked or not.
+   * - **Initial inset from corner (LH side)**
+     - ``INSET``
+     - [number]
+
+       Default: 0.0
+     - Offset the points relative to the upper left corner.
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/vectorcreation.rst]: Both are OK.

File: source/docs/user_manual/processing_algs/qgis/vectorcreation.rst [hunk 11]
```diff
@@ -432,72 +822,152 @@ A minimum distance can be specified, to avoid points being too close to each oth
...
 
 
 .. _qgisrandompointsinsidepolygons:
 
 Random points inside polygons
 -----------------------------
-Creates a new point layer with a given number of random points, all of them within
-a given layer.
+Creates a new point layer with a given number of random points
+inside each polygon of the input polygon layer.
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/vectorcreation.rst]: It is defined when the algorithm is run.  We are talking about *the* polygon layer that the user has specified as "Input layer".

File: source/docs/user_manual/processing_algs/qgis/vectorcreation.rst [hunk 12]
```diff
@@ -506,33 +976,72 @@ A minimum distance can be specified, to avoid points being too close to each oth
...
+       * 0 --- Points count: number of points for each feature
+       * 1 --- points density: density of points for each feature
+
+   * - **Number or density of points**
+     - ``VALUE``
+     - [expression]
+
+       Default: 1.0
+     - The number or density of points, depending on the chosen
+       ``Sampling strategy``.
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/vectorcreation.rst]: It is nice to be able to point to elements of the user interface. I have changed to guilabel.

File: source/docs/user_manual/processing_algs/qgis/vectorcreation.rst (hunk 6 above)
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/vectorcreation.rst]: 0.0

**Comment by havatv** [comment]
> A quick review. Sorry. I won't be that available this week.

It may have been quick, but you found many errors.  Thanks for helping!

**Pull request merged**